packages/engine-core/src/data-proxy/utils/getClientVersion.ts

```diff
@@ -28,7 +25,7 @@ export function getClientVersion(config: EngineConfig) {
   }
 
   // if it's an integration version, we resolve its data proxy
-  if (suffix === 'integration' && semverRegex.test(version)) {
+  if (suffix === 'integration' || clientVersion === '0.0.0') {
```

Comment:
Where does the 0.0.0 case trigger?

Reply:
In our local tests or but really when you develop locally. I don't think we need to check the regex again, integration versions will have a valid one always.